rust/processor/src/db/common/models/token_v2_models/parquet_v2_token_datas.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+// PK of current_token_datas_v2, i.e. token_data_id
+pub type CurrentTokenDataV2PK = String;
+
+#[derive(Clone, Debug, Deserialize, FieldCount, Serialize)]
+pub struct TokenDataV2 {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub token_data_id: String,
+    pub collection_id: String,
+    pub token_name: String,
+    pub maximum: Option<u64>,
```

Comment:
i thought maximum, supply, decimal were moved to fungible_asset_metadata in https://github.com/aptos-labs/aptos-indexer-processors/pull/354

`transaction_timestamp` -> `block_timestamp`

where did `is_deleted_v2` get moved to?